zeppelin-web/src/components/baseUrl/baseUrl.service.js

```diff
@@ -24,8 +24,8 @@ function baseUrlSrv() {
       }
     }
     //Exception for when running locally via grunt
-    if (port === 3333 || port === 9000) {
```

Comment:
Yes, as far as i know this port is old web development mode. Thus we don't need `3333` port :)